packages/flutter/lib/analysis_options_user.yaml

```diff
@@ -26,9 +26,6 @@ analyzer:
   errors:
     # treat missing required parameters as a warning (not a hint)
     missing_required_param: warning
-    # treat missing returns as a warning (not a hint)
-    # TODO(ianh): https://github.com/flutter/flutter/issues/20114
-    missing_return: ignore
```

Comment:
I think you instead want to delete the todo, and change `missing_return` from ignore to warning (if you're trying to restore the pre-dart sdk roll behavior).